ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+### Flow
+
+* [ ] Bug
+    * [ ] Can you reproduce the problem in [safe mode](https://atom.io/docs/latest/hacking-atom-debugging#check-if-the-problem-shows-up-in-safe-mode)?
+    * [ ] Did you check the [debugging guide](https://atom.io/docs/latest/hacking-atom-debugging)?
+    * [ ] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
+    * [ ] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
+    * [ ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if it is already reported?
+    * For more information on how to write a good bug report [look at the CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report).
+* [ ] Enhancement
+    * [ ] Are you running the [latest version of Atom](https://atom.io/docs/latest/hacking-atom-debugging#update-to-the-latest-version)?
+    * [ ] Did you check to see if [a package](https://atom.io/packages) already offers the feature?
+    * [ ] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
+    * [ ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if it is already requested?
+    * For more recommendations on how to write a good enhancement suggestion [see the CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion).
+
+### Description
+
+[Description of the bug or feature]
+
+### Repro Steps
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+**Expected:** [What you expected to happen]
```

Comment:
I think an extra line break is needed after this line, or else the Expected and Actual will display on the same line.